src/components/filters/media-type-filter/index.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2019 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import React, { useEffect, useState } from 'react'
+import T from 'i18n-react/dist/i18n-react';
+import MediaUploadTypeInput from '../../inputs/media-upload-type-input';
+import Select from 'react-select'
+
+import styles from './index.module.less'
+
+const MediaTypeFilter = ({ onChange, value, id, summitId }) => {
+
+    const [hasOrNotHasValue, setHasOrNotHasValue] = useState(value.filter === 'has_media_upload_with_type' ? { label: 'Has', value: true } : { value: 'Not Has', value: false });
```

Comment:
please review this comparing with https://github.com/fntechgit/summit-admin/blob/57889f4a946e912ae66d2250f37586d176366cb3/src/components/inputs/email-filter/index.js#L14